frontend/src/components/dragHandle.tsx

```diff
@@ -0,0 +1,67 @@
+import {MouseEvent as ReactMouseEvent, useEffect, useRef, useState} from "react";
+import styled from "styled-components";
+
+const Handle = styled.div`
+  position: absolute;
+  
+  box-shadow: none;
+  background: none;
+
+  user-select: none;
+`;
+
+export const DragHandle = (props: {
+  className?: string,
+  onMove: (vector: DOMPoint) => void,
+  children: React.ReactNode,
+}) => {
+  const { className, onMove, children } = props;
+
+  const [dragging, setDragging] = useState(false);
+  const positionRef = useRef(new DOMPoint(0, 0));
+
+  const onMouseUp = (event: MouseEvent) => {
+    setDragging(false);
+    console.log("mouseup");
```

Comment:
console

Reply:
Keeping since needed to reserve event for future.